src/Core/src/Hosting/MauiHostEnvironment.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Maui.ApplicationModel;
+using Microsoft.Maui.Storage;
+
+namespace Microsoft.Maui.Hosting
+{
+	public class MauiHostEnvironment : IHostEnvironment
+	{
+		public string EnvironmentName
+		{ 
+			get
+			{
+#if DEBUG
```

Comment:
Yeahhhhh this obviously doesn't work for the client build configuration. So either we find another way, or it will just be hardcoded

Reply:
I'm not sure if what I'm about to say is a great idea, but one idea I had was to go grab the current Application's Assembly and check it's `assembly: AssemblyConfigurationAttribute` for `Debug` or `Release`. That would at least work to change between Development and Production, where here the user will always get "Production" since they will never have Debug builds.